pages/_app.tsx

```diff
@@ -28,9 +28,7 @@ const chains: Chain[] = [{ ...chain.rinkeby, name: 'Rinkeby' }];
 const wallets = getDefaultWallets({
   chains,
   appName: 'Backed',
-  jsonRpcUrl: ({ chainId }) =>
-    chains.find((x) => x.id === chainId)?.rpcUrls?.[0] ??
-    chain.mainnet.rpcUrls[0],
+  jsonRpcUrl: process.env.NEXT_PUBLIC_JSON_RPC_PROVIDER!,
```

Comment:
```suggestion
  infuraId: process.env.NEXT_PUBLIC_INFURA_ID,
  jsonRpcUrl: process.env.NEXT_PUBLIC_JSON_RPC_PROVIDER!,
  
```

Reply:
Discussed reasons why this is happening in discord, this should resolve until next release this week.